lib/ui.js

```diff
@@ -95,7 +96,7 @@ exports.log = function(type, msg) {
     msg = format[type](msg);
 
   if (logging) {
-    if ((logLevel === 'debug') ||
+    if ((logLevel === DEFAULT_LOG_LEVEL) ||
```

Comment:
this is so any messages without a `type` get shown only when the log level is at the default